server/db/models/product.js

```diff
@@ -0,0 +1,26 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Product = db.define('product', {
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false
```

Comment:
@garrett-green We can review this is the stand up, but I think we need both option: allowNull and not: empty, that to avoid having someone not defining the name, or if someone try to change it to an empty string '', it won't accpet it.